src/asserts/object-assert.js

```diff
@@ -0,0 +1,28 @@
+const { Violation } = require('validator.js');
+const { isFunction, isObject } = require('lodash');
+
+/**
+ * Export `ObjectAssert`.
+ */
+
+module.exports = function objectAssert() {
```

Comment:
I'm not an expert on TS but I don't thing that we want to do this as it would allow for things such as arrays. The `Record`s type prototype seems to be a plainObject. Can't we simply do something along these lines:

```js
module.exports = function plainObjectTSAssert() {
  /**
   * Class name.
   */

  this.__class__ = 'plainObjectTS';

  /**
   * Validation algorithm.
   */

  this.validate = value => {
    const prototype = Object.getPrototypeOf(value);

    if (!isPlainObject(prototype)) {
      throw new Violation(this, value);
    }
    
    return true;
  };

  return this;
};
```